L1MenuCheck_FromGT.sh

```diff
@@ -28,17 +28,26 @@ xmlhash=`conddb --db "oracle+frontier://@frontier%3A%2F%2F%28proxyurl%3Dhttp%3A%
 
 #All this is to get around the fact that conddb dump tries to modify the release base
 #I have tried to make this robust, but I am not very confident. I have no doubt that this will break sometime in the future...
-srcpath=`echo $CMSSW_SEARCH_PATH | sed '0,/:.*$/s/:.*$//'`
+cmsswbase=${CMSSW_BASE}
```

Comment:
why don't you use `$CMSSW_BASE` directly, instead of assigning it to `cmsswbase` ?

Reply:
i think this is because, below, a temporary local area is created and initialized ([here](https://github.com/cms-tsg-fog/hltpro/pull/19/files#diff-217e7cc897d0a82a12a9eec26bc2c34aR42)), thus changing the original value of `$CMSSW_BASE`